.travis.yml

```diff
@@ -1,7 +1,9 @@
 language: ruby
 
 rvm:
-  - 2.1.1
+  - 2.3.1
+  - 2.2.6
+  - 2.1.10
   - 2.0.0
```

Comment:
We could replace with : 

```
- 2.3.1
- 2.2
- 2.1
- 2.0
```

so it would use `2.3.1` and the latest version of the `2.0.X`, 2.1.X` and `2.2.X` branches.